src/scripts/itemDiscuss/dimItemDiscuss.directive.html

```diff
@@ -79,4 +79,4 @@
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
I'll check, I just switched from atom over to VS Code today, so maybe I forgot to set up editorconfig